lib/routes/uestc/bbs.ts

```diff
@@ -0,0 +1,95 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { parseDate } from '@/utils/parse-date'; // 解析日期的工具函数
+import timezone from '@/utils/timezone';
+import { config } from '@/config';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/bbs/:types?',
+    name: '清水河畔',
+    maintainers: ['huyyi'],
+    categories: ['bbs'],
+    example: '/uestc/bbs/newthread',
+    parameters: { types: '选择内容类型(多选`,`分割），可选值：[newreply,newthread,digest,life,hotlist]。默认为所有。' },
+    features: {
+        requireConfig: [
+            {
+                name: 'UESTC_BBS_COOKIE',
+                optional: false,
+                description: '河畔的cookie',
+            },
+            {
+                name: 'UESTC_BBS_AUTH_KEY',
+                optional: false,
+                description: '河畔Header中的authorization字段',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    description: `
+:::tip
+仅支持自建，您需要设置以下配置才能正常使用：
+-   河畔cookie: \`UESTC_BBS_COOKIE\`
+-   Header中的授权字段: \`UESTC_BBS_AUTH_KEY\`
+:::
+`,
+    radar: [
+        {
+            source: ['bbs.uestc.edu.cn/*'],
+            target: '/uestc/bbs/newthread',
+        },
+    ],
+    handler: async (ctx) => {
+        const { bbs_cookie, bbs_auth_str } = config.uestc;
+        if (!bbs_cookie || !bbs_auth_str) {
+            throw new Error('未配置 Cookie 或 Authorization。请检查配置。');
```

Comment:
Please throw https://github.com/DIYgod/RSSHub/blob/b7a8eb3da6f9b9157c327afaad8f465d1826d5d0/lib/errors/types/config-not-found.ts#L5 instead